### PR TITLE
Final prod touches

### PR DESCRIPTION
**Pull request by lewiskyron**
## Description
This PR comments out different segments of our code that are not currently being used. This is with the intention of not having elements that are not functional in production.

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Tests (adding or updating tests)
- [ ] Documentation update

## Testing Instructions
Check into the branch and you should notice. 
- The `login` and `signup` page now only have google sign-in
- The notifications button on the `navbar` is removed(this is not functional yet). 
- Tags have been added. They are now displayed on when projects are displayed on the home screen.

## Screenshots (if applicable)

If your changes include visual components, add screenshots showing the affected pages or elements.

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -15,8 +16,17 @@ const Card: React.FC<CardProps> = ({
 	date,
 	descriptionShort,
 	headline,
+	tags = [], // Default empty array
 	onExploreMore,
 }) => {
+	// Define darker colors for tags
+	const tagColors: Record<string, string> = {
```
Comment (FlaIespa) [inline src/components/Card.tsx:23]: nice touch with adding the tags!

File: src/components/Card.tsx [hunk 2]
```diff
@@ -109,6 +120,33 @@ const Card: React.FC<CardProps> = ({
 				</Typography>
 			</Box>
 
+			{/* Tags Section (Below Description) */}
```
Comment (FlaIespa) [inline src/components/Card.tsx:123]: Nice! We can remove the comments later.

**Review by FlaIespa — CHANGES_REQUESTED** [review]
Hey, Kyron! Great work here. I haven't approved it yet because the Vercel deployment is failing. @ucey-star given that you have access to the deployment in vercel, maybe is something you can check. 
Also, in the login page we can see that is rendering with a small error. 
Screenshot 2024-11-19 at 5 19 42 PM

File: src/components/Card.tsx (hunk 2 above)
Comment (lewiskyron) [reply src/components/Card.tsx:123]: sure noted.

**Review by FlaIespa — APPROVED** [review]

**Pull request merged**